app/controllers/concerns/sufia/file_sets_controller_behavior.rb

```diff
@@ -28,7 +28,7 @@ module FileSetsControllerBehavior
     end
 
     def new
-      @upload_set_id = ActiveFedora::Noid::Service.new.mint
+      @upload_set_id = SecureRandom.uuid
```

Comment:
Given the size of the UUID identifier space this is very unlikely buuuuut couldn't this line generate a UUID already allocated in Fedora? Do we worry about that, and hook into AF here? Or add a method to AF to spit out an unused UUID?